.github/workflows/tests.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         run: brew services start postgresql
 
       - name: Enable PostGIS extension
-        run: psql -c "CREATE EXTENSION postgis;" postgres
+        run: psql -h 127.0.0.1 -U runner -c "CREATE EXTENSION postgis;" postgres
```

Comment:
If you are finding problems with macOS but postgis is known to just work with that platform maybe we can avoid running these tests?

Also note that creating the extension from pytest rather than from runner would probably need no configuration tweaking

Reply:
Yes, and installing it seems quite heavy, maybe the best solution is to test it only on Linux (either using the postgis image or setting it up somehow in the unit test itself)

Reply:
If postgis image and Linux work, I'm happy with that :)

Reply:
Removed, now it's only on linux (where it's already configured in the postgis image so no CREATE EXTENSION step is needed)